web_auto_refresh/readme.rst

```diff
@@ -0,0 +1,56 @@
+web auto refresh
```

Comment:
this is my  first time use github, only by web interface, I just copied another readme file, then do the modification as necessary, in the web interface, seems there is no way to format the content.
I will try to use the windows desktop tool to update the content,